pyproject.toml

```diff
@@ -57,7 +57,7 @@ full = [
     "dissect.fat>=3,<4",
     "dissect.ffs>=3,<4",
     "dissect.fve>=4,<5; platform_system != 'Windows' or platform_python_implementation != 'PyPy'",
-    "dissect.jffs>=1,<2",
+    "dissect.jffs>=1.5.dev,<2",     # TODO: update on release 
```

Comment:
I assume you've created a ticket for this (internally) or do you wish to wait for a release?